docs/guide/getting-started.md

```diff
@@ -99,6 +99,15 @@ end
 
 _Note: Content can't be passed to a component via a block in controllers. Instead, use `with_content`. In versions of Rails < 6.1, rendering a ViewComponent from a controller doesn't include the layout._
 
+When using [turbo-rails](https://github.com/hotwired/turbo-rails) be sure to set `content_type` as `text/html`:
+
+```ruby
+# app/controllers/home_controller.rb
+def create
+  render(ExampleComponent.new, content_type: "text/html")
```

Comment:
Does this work?

```suggestion
  render html: ExampleComponent.new
```

Reply:
@Spone this made the response content type "text/html", which is right, but the response will be a string with the `ViewComponent` instance, like:

- Controller:
```
class ExampleController < ApplicationController
  def create
    render html: ExampleComponent.new
  end
end
```

- Response:
```html
#<ExampleComponent:0x00007fa2f7bcf150>
```

Reply:
You're right! That a bit unexpected for users, I wonder if there's a way to improve that from ViewComponent side.